Source/WebCore/dom/ChildListMutationScope.cpp

```diff
@@ -57,76 +56,74 @@ ChildListMutationAccumulator::~ChildListMutationAccumulator()
 {
     if (!isEmpty())
         enqueueMutationRecord();
-    accumulatorMap().remove(m_target.ptr());
+    accumulatorMap().remove(m_target.get());
 }
 
 Ref<ChildListMutationAccumulator> ChildListMutationAccumulator::getOrCreate(ContainerNode& target)
 {
-    AccumulatorMap::AddResult result = accumulatorMap().add(&target, nullptr);
-    RefPtr<ChildListMutationAccumulator> accumulator;
-    if (!result.isNewEntry)
-        accumulator = result.iterator->value;
-    else {
-        accumulator = adoptRef(new ChildListMutationAccumulator(target, MutationObserverInterestGroup::createForChildListMutation(target)));
-        result.iterator->value = accumulator.get();
-    }
-    ASSERT(accumulator);
-    return accumulator.releaseNonNull();
+    RefPtr<ChildListMutationAccumulator> result;
```

Comment:
I think this is not quite the right name. This is “newly created accumulator” rather than “the function’s result”, which could be an already-existing accumulator.